Week3/homework/src/createToDoItem.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function createToDoItem(connection, errorHandler, req, res) {
+  const username = req.params.username;
+  const password = req.params.password;
+  const listDescription = req.params.listDescription;
+  const toDoDescription = req.params.toDoDescription;
+  const queryString1 = 'SELECT user_id FROM users WHERE user_name = ? AND user_password = ?';
```

Comment:
**Non-blocking:**

I see here and there you retrieve user as the first step in your code. And it's a repetitive action.
There is DRY rule: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

**I'm saying so because** of your colleagues will tell you this as well. Better be prepared.